### PR TITLE
for #1147 clarify in docs to set the VAULT_ADDR env var

### PR DESCRIPTION
**Pull request by lhagemann**
just a quick note to clarify the fix for 

```
Error initializing Vault: Put https://127.0.0.1:8200/v1/sys/init: http: server gave HTTP response to HTTPS client
```

by adding the `VAULT_ADDR` env var

### CODE REVIEW

**Comment by wurde** [comment]
There is an additional scenario where if you use the Vault CLI using "sudo" the client will use HTTPS and receive the above error message. This isn't documented anywhere and was a huge pain.

The ~/.vault-token is written with root ownership requiring, which makes all non-sudo commands to receive `permission denied` errors. Then when you try to use "sudo" you get `server gave HTTP response to HTTPS client`. It is a circle of undocumented pain.

File: website/source/intro/getting-started/deploy.html.md [hunk 1]
```diff
@@ -110,6 +110,13 @@ are created, and the initial root token is setup. To initialize Vault
 use `vault init`. This is an _unauthenticated_ request, but it only works
 on brand new Vaults with no data:
 
+The vault client defaults to using `https` scheme, if you are running vault locally without a proper SSL cert, you'll want to set the client address to not use `https`:
+
+```
+export VAULT_ADDR=http://127.0.0.1:8200
+```
+prior to intializing the vault server.
```
Comment (pschmitt) [inline website/source/intro/getting-started/deploy.html.md]: There's a typo here. It should read "initializing"

**Comment by kenden** [comment]
@ghost, I created a bug for your "sudo / .vault-token issue.

The PR looks good.

**Comment by kalafut** [comment]
This guidance has since been added to the explanation of the `listener` parameter: https://www.vaultproject.io/intro/getting-started/deploy.html#listener

**Pull request closed without merging**